addons/mail/static/src/models/activity_view/activity_view.js

```diff
@@ -63,13 +68,88 @@ registerModel({
         onClickUploadDocument() {
             this.fileUploaderView.component.openBrowserFileUploader();
         },
+        /**
+         * @private
+         * @returns {String|FieldCommand}
+         */
+        _computeAssignedUserText() {
+            if (!this.activity.assignee) {
+                return clear();
+            }
+            return _.str.sprintf(this.env._t("for %s"), this.activity.assignee.nameOrDisplayName);
+        },
+        /**
+         * @private
+         * @returns {String|FieldCommand}
+         */
+        _computeDelayLabel() {
+            if (!this.activity.dateDeadline) {
+                return clear();
+            }
+            const today = moment().startOf('day');
+            const momentDeadlineDate = moment(auto_str_to_date(this.activity.dateDeadline));
+            // true means no rounding
+            const diff = momentDeadlineDate.diff(today, 'days', true);
+            if (diff === 0) {
+                return this.env._t("Today:");
+            } else if (diff === -1) {
+                return this.env._t("Yesterday:");
+            } else if (diff < 0) {
+                return _.str.sprintf(this.env._t("%d days overdue:"), Math.abs(diff));
+            } else if (diff === 1) {
+                return this.env._t("Tomorrow:");
+            } else {
+                return _.str.sprintf(this.env._t("Due in %d days:"), Math.abs(diff));
+            }
+        },
         /**
          * @private
          * @returns {FieldCommand}
          */
         _computeFileUploaderView() {
             return this.activity.category === 'upload_file' ? insertAndReplace() : clear();
         },
+        /**
+         * @private
+         * @returns {String|FieldCommand}
+         */
+        _computeFormattedCreateDatetime() {
+            if (!this.activity.dateCreate) {
+                return clear();
+            }
+            const momentCreateDate = moment(auto_str_to_date(this.activity.dateCreate));
+            const datetimeFormat = getLangDatetimeFormat();
+            return momentCreateDate.format(datetimeFormat);
+        },
+        /**
+         * @private
+         * @returns {String|FieldCommand}
+         */
+        _computeFormattedDeadlineDate() {
+            if (!this.activity.dateDeadline) {
+                return clear();
+            }
+            const momentDeadlineDate = moment(auto_str_to_date(this.activity.dateDeadline));
+            const datetimeFormat = getLangDateFormat();
+            return momentDeadlineDate.format(datetimeFormat);
```

Comment:
same remark

Reply:
This one is also just formatting a date to human format.